docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+services:
+  mongo:
+    image: mongo
+    ports:
+    - "27017:27017"
+  node:
+    build: .
+    ports:
+      - "3000:3000"
+    working_dir: /shellshare
+    environment:
+      - NODE_ENV=production
```

Comment:
Add `MONGOLAB_URI=mongodb://mongo:27017/shellshare`

Reply:
Done! For some reason I didn't manage to get that working and I had changed the `config.js`. But it works now :)